main.css

```diff
@@ -0,0 +1,49 @@
+body {
+    background-color: lightgreen;
+  }
+
+p {
+    font-family: "Titillium Web", sans-serif;
+    text-align: center;
+  }
+
+h1 {
+    font-family: "IBM Plex Mono", monospace;
+    text-align: center;
+  }
+  
+h2 {
+    font-family: "IBM Plex Mono", monospace;
+    text-align: center;  
+  
+#submit-button, {
```

Comment:
ln 19, remove the comma after #submit-button and your styling should work.

Reply:
awww, that's why. Thanks!